scripts/set-default-credentials.sh

```diff
@@ -9,10 +9,14 @@ if [ "${TRAVIS}" != "true" ]; then
   export TRAVIS_REPO_SLUG="firebase/firebase-tools"
 fi
 
-GOOGLE_APPLICATION_CREDENTIALS="${CWD}/scripts/creds-private.json"
-if [ "${TRAVIS_REPO_SLUG}" == "firebase/firebase-tools" ]; then
-  GOOGLE_APPLICATION_CREDENTIALS="${CWD}/scripts/creds-public.json"
+if [ "${LOCAL}" != "true" ]; then
+  GOOGLE_APPLICATION_CREDENTIALS="${CWD}/scripts/creds-private.json"
+  if [ "${TRAVIS_REPO_SLUG}" == "firebase/firebase-tools" ]; then
+    GOOGLE_APPLICATION_CREDENTIALS="${CWD}/scripts/creds-public.json"
+  fi
+  export GOOGLE_APPLICATION_CREDENTIALS
+else
+  echo "Not setting GOOGLE_APPLICATION_CREDENTIALS because LOCAL=${LOCAL}"
```

Comment:
Why was this necessary?

Reply:
Makes it much easier to run these integration tests locally and re-use whatever default credentials the CLI already has.